README.md

```diff
@@ -1,2 +1,117 @@
-# physical-collectibles-nft-global-event-website
-Global Event dedicated to leading Blockchain technologies in space of producing physical collectibles.
+# Microverse Capstone Project
+## Module 1
+
+Build a Custom Website based on Template.
+
+The subject chosen: NFT Global Event Presentation. Includes custom design created for this project.
+
+
+
+## Built With
+
+- CSS HTML
+- Git version control system, Github workflows
+- javascript
+
+
+
+## Live Demo (if available)
```

Comment:
Your live link is functional! Well done using GitHub Pages to deploy your website :bomb: